kubernetes_state/datadog_checks/kubernetes_state/kubernetes_state.py

```diff
@@ -96,6 +96,23 @@ def check(self, instance):
         for job_tags, job_count in iteritems(self.job_failed_count):
             self.monotonic_count(scraper_config['namespace'] + '.job.failed', job_count, list(job_tags))
 
+    def _filter_metric(self, metric, scraper_config):
+        if scraper_config['telemetry']:
+            # name is like "kube_pod_execution_duration"
+            name_part = metric.name.split("_", 3)
+            family = name_part[1]
```

Comment:
Is this safe? 🤔

Reply:
ksm metrics always have the same parttern. But I can add another check on the size